### PR TITLE
add static site generator

### LINKED ISSUE #2: Website mockup
I made a website mockup. You should be able to see it rendered [here]. I've also included some screenshots at the bottom of this comment.

All kind of feedback is welcomed.

If you are all happy with it I can make a PR setting up a static site generator.

[here]: https://japaric.github.io/stunning-waffle/index.html

Some notes:

- The background color is the color of the WG logo.

- Images and videos always have 4:3 aspect ratio.

- Videos should be .mp4; Chrome doesn't like .webm. (Firefox for Android seems to prefer .webm though)

- Width of images and videos should not be greater than 900px (there is no benefit in having larger images because they'll be shrunk)

- Project description should be short or the text will overflow with some resolutions; 1 paragraph with a max of 280 characters (tweet size) is about the limit

- I've tested that the site behaves the same on Firefox and on Chrome, and I doubt that it displays correctly in IE.

- I'm open to suggestions about what the menu should contain (currently it has HOME, ABOUT and SUBMIT)

- The project names are anchor links, e.g. [anchor to the second project](https://japaric.github.io/stunning-waffle/index.html#lorem-ipsum)

- The author names should link to their website / GH profile page.

- LEARN MORE should link to the project website / GH repository.

- Clicking an image should take you to the project website / GH repository. However, videos don't have that behavior otherwise the video controls wouldn't work.

### Mobile view

screen shot 2018-11-21 at 16 54 47

### Desktop view

[Full page image](https://i.imgur.com/mZVwA6e.png)

### PR DESCRIPTION
**Pull request by japaric**
content is to be entered in data.yml

closes #2

r? @rust-embedded/resources

### CODE REVIEW

File: css/main.css [hunk 1]
```diff
@@ -0,0 +1,265 @@
+/* Style based on the NEDERBURG Hugo theme (https://themes.gohugo.io/hugo-nederburg-theme/) */
```
Comment (thejpster) [inline css/main.css]: The licence (https://github.com/appernetic/hugo-nederburg-theme/blob/master/LICENSE) requires that the copyright statement and licence text in included in all copies of substantial portions of the software. A link is not sufficient.

**Review by thejpster — CHANGES_REQUESTED** [review]

**Comment by adamgreig** [comment]
LGTM, aside from @thejpster's issue.

File: css/main.css (hunk 1 above)
Comment (japaric) [reply css/main.css]: > included in all copies of substantial portions of the software

I have added a copy of the nederburg license. I should note that I implemented this file from scratch by inspecting the demo website (I can't really read complex CSS files). I think it still qualifies as derivative work though.

**Comment by japaric** [comment]
@adamgreig @thejpster reminder to re-r? this PR

**Review by adamgreig — APPROVED** [review]

**Comment by japaric** [comment]
bors r=adamgreig

**Comment by japaric** [comment]
The bors build passed (https://travis-ci.org/rust-embedded/showcase/builds/478698501) but the branch protection settings were off so I'm going to merge this manually and then fix the branch settings.

**Pull request merged**